README.md

```diff
@@ -153,6 +153,13 @@ feed:
     - updates
 ```
 
+Or, to generate a feed for all categories:
```

Comment:
We might want to explain that in the current implementation, this means that each collection will get its own category feed. This operates differently than tag-based feeds where they unify all documents from any collection into a single feed.